inc/admin_config_orders.php

```diff
@@ -133,6 +142,32 @@ public function beforeUpdate($new_data, $old_data, $id)
 					}
 				}
 			}
+
+			// Check for changes and add to the log
+			$now = time();
+			foreach ($new_data as $key => $value) {
+				$oldval = $old_data[$key];
+				if ($value !== $oldval && array_key_exists($key, $this->fields))
+				{
+					$title = $this->fields[$key]['title'];
+
+					if ($key == 'order_status')
+					{
+						$value = vstore::getStatus($value);
+						$oldval = vstore::getStatus($oldval);
+					}
+
+					$log[] = array(
+						'datestamp' => $now,
+						'user_id' => USERID,
+						'user_name' => USERNAME,
+						'text' => sprintf('Changed %s from "%s" to "%s".', $title, $oldval, $value)
```

Comment:
Please check out $tp->lanVars(). You can use "Changed [x] from [y] to [z]" which should have less issues when moved to language files.